de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/simulator/JSSimulatorNodeModel.java

```diff
@@ -96,28 +94,36 @@ public ValidationError validateViewValue(JSSimulatorViewValue viewContent) {
   }
 
   @Override
-  public void saveCurrentValue(NodeSettingsWO content) {}
+  public void saveCurrentValue(NodeSettingsWO content) {
+  }
 
   @Override
   public JSSimulatorViewValue getViewValue() {
-    JSSimulatorViewValue val;
+
+    JSSimulatorViewValue value = super.getViewValue();
     synchronized (getLock()) {
-      val = super.getViewValue();
 
-      if (val == null) {
-        val = createEmptyViewValue();
+      if (value.getSimulations().length == 0) {
+        copyConfigToView(value);
       }
 
-      if (val.simulations == null && port != null && port.simulations != null) {
+      if (value.getSimulations() == null) {
         // Convert from FskSimulation(s) to JSSimulation(s)
-        final List<Parameter> parameters = SwaggerUtil.getParameter(port.modelMetadata);
-        val.simulations = port.simulations.stream().map(it -> toJSSimulation(it, parameters))
-            .collect(Collectors.toList());
-        val.selectedSimulationIndex = port.selectedSimulationIndex;
+        value.setSimulations(convertSimulations(port.modelMetadata));
+        value.setSelectedSimulationIndex(port.selectedSimulationIndex);
+       
+      }
+
+      if (value.getModelMath() == null) {
+        try {
+          value.setModelMath(FromOjectToJSON(SwaggerUtil.getModelMath(port.modelMetadata)));
+        }catch(JsonProcessingException e) {

```

Comment:
We need a space between closing brace and catch, and another between catch and the opening parentheses https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace